### PR TITLE
Sentry: Add annotations to service

### PR DESCRIPTION
**Pull request by elordahl**
#### What this PR does / why we need it:
Adding annotations for the Service allows for improved customization of the service.  Specifically, the provisioned load balancer (when using default `type: LoadBalancer`) can support settings like cross-zone load balancing and internal-only exposure, among many others, in AWS.

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

@rothgar

### CODE REVIEW

File: stable/sentry/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Sentry is a cross-platform crash reporting and aggregation platform.
 name: sentry
-version: 0.4.1
+version: 0.4.2
```
Comment (cpanato) [inline stable/sentry/Chart.yaml]: it is a new functionality please bump to 0.5.0

Comment (elordahl) [reply stable/sentry/Chart.yaml]: done!

**Comment by elordahl** [comment]
what is this waiting on?

**Pull request merged**